apps/air-discount-scheme/api/jest.config.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
-  preset: '../../../jest.preset.js',
-  coverageDirectory: '../../../coverage/apps/air-discount-scheme/api',
-  setupFiles: ['./test/environment.jest.ts'],
-  globals: { 'ts-jest': { tsconfig: '<rootDir>/tsconfig.spec.json' } },
+  preset: './jest.preset.js',
+  rootDir: '../../..',
+  roots: [__dirname],
+  coverageDirectory: '<rootDir>/coverage/apps/air-discount-scheme/api',
+  setupFiles: [`${__dirname}/test/environment.jest.ts`],
+  globals: { 'ts-jest': { tsconfig: `${__dirname}/tsconfig.spec.json` } },
```

Comment:
Can you explain why we need to move the root?

This will definitely conflict with future NX migrations (not a big issue though).

Reply:
Worse yet ... new apps/libs will not be correctly configured...

Do we need to create our own templates or lint this somehow?

Reply:
Moving the root so jest sees and generates code coverage across workspaces/projects. Not sure how these come to be but yes - we need future ones to be similar.